core/modules/layout/plugins/context/entity_layout_context_handler.inc

```diff
@@ -42,4 +39,42 @@ class EntityLayoutContext extends LayoutContext {
       return parent::label();
     }
   }
+
+  function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
+
+    $form['entity'] = array(
+      '#title' => t('Enter the title or ID of a @entity entity', array('@entity' => $this->plugin)),
+      '#type' => 'textfield',
+      '#maxlength' => 512,
+      '#autocomplete_path' => 'path-autocomplete',
+      '#weight' => -10,
+    );
+  }
+
+  /**
+   * Submit handler for the settings form for this context item.
+   */
+  function formSubmit($form, &$form_state) {
+    $excluded = backdrop_map_assoc(array('cancel', 'submit', 'form_build_id', 'form_token', 'form_id', 'op', 'add', 'context'));
+    $settings = array_diff_key($form_state['values'], $excluded);
+    $this->name = $this->plugin . $settings['entity'];
+    // @todo deal with name changes.
+    $this->settings = array_merge($this->settings, $settings);
+  }
+
+  /**
+   * Return a summary for this context.
+   */
+  function getAdminSummary($path = NULL) {
+    if ($this->position) {
+      $argument_parts = explode('/', $path);
+      $argument_parts[$this->position] = '<strong>' . $argument_parts[$this->position] . '</strong>';
+
+      return t('Position @index:', array('@index' => $this->position + 1)) . ' ' . implode('/', $argument_parts);
+    }
+    else {
+      return t('Special context: @id', array('@id' => $this->settings['entity']));
```

Comment:
We should make sure that whatever we pass to `@id` exists. I got this in the PR sandbox:

![Screen Shot 2020-04-28 at 12 47 15 pm](https://user-images.githubusercontent.com/2423362/80473154-68fbac80-894e-11ea-85d8-8495c73e9818.png)

Reply:
fixed